cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -588,7 +657,7 @@ private String subQuery(String index, CQLTermNode node, Table correlation) throw
     boolean indexInTable =  childParentForeignKey == null;
     ForeignKeys fkey = indexInTable ? findForeignKey(correlation, PK_COLUMN_NAME, dbTable) : childParentForeignKey;
     Table indexTable = indexInTable ? dbTable : correlation;
-    
+
```

Comment:
Can you remove this unused method `subQuery`?

Reply:
done